Frontend/ui-driver/src/Services/Backend.purs

```diff
@@ -1391,4 +1391,14 @@ getReelsVideo reelsKey language = do
   headers <- getHeaders "" false
   withAPIResult (EP.getReelsData reelsKey language) unwrapResponse $ callAPI headers (GetAllReelsVideosReq reelsKey language)
   where
-    unwrapResponse x = x
\ No newline at end of file
+    unwrapResponse x = x
+    
+----------------------------------------------------------------------- specialLocation list -------------------------------------------------------------------------------
+
+getSpecialLocationListBT ::  SpecialLocationFullReq -> FlowBT String SpecialLocationFullRes
+getSpecialLocationListBT req = do
```

Comment:
makeSure that in case of API failure it doesn't do any unexpected behaviour.

Reply:
back pointer is set

Reply:
For this we should have used callAPIBT